README.md

```diff
@@ -1,7 +1,7 @@
 <a name="readme-top"></a>
 
 <div align="center">
-  <img src="img/fbprofilepic.jpg" alt="logo" width="50"  height="auto" />
+  <img src="img/archesportslogo.jpg" alt="logo" width="50"  height="auto" />
   <br/>
 </div>
```

Comment:
- [ ] Nice work adding some of the sections on the [README](https://github.com/microverseinc/readme-template), however, you're meant to add all sections with just `FAQ` being optional. I've attached screenshots below to show you what to add. This makes your project possess all sections that make it look professional 

## Expected README  Header:
![image](https://user-images.githubusercontent.com/50990925/226778810-21f02d49-388c-44c6-bdca-bb8f80c51658.png)